.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Rust Format Check
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  fmt:
+    name: fmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
```

Comment:
Just realized we don't need this for stable? `cargo` should be available out of the box.